### ISSUE TITLE
Request: Chrome extensions

### ISSUE THREAD

**Issue by NikilRagav** [agata/dl.biscuit#10]
I use Boomerang with my emails to see how likely someone is to respond to my emails. I also use ad block. 

I would love to have these features built into Biscuit as well or give the ability to add any extensions from Chrome webstore

**Comment by JosiasVasquez** [comment]
Yes, I would like chrome extensions too accessibility, add blocker etc.
